qfdmo/views/adresses.py

```diff
@@ -587,6 +587,10 @@ def refresh_acteur_view(request):
 
 @require_GET
 def get_object_list(request):
+    q = request.GET.get("q")
```

Comment:
```suggestion
    query = request.GET.get("q")
```

je comprends le `q` dans l'URL, à la limite, mais dans notre code on peut éviter la *one-letter variable* ?